cli/cmd/user/add.go

```diff
@@ -0,0 +1,82 @@
+package user
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/rilldata/rill/cli/cmd/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func AddCmd(cfg *config.Config) *cobra.Command {
+	var orgName string
+	var projectName string
+	var email string
+	var role string
+
+	addCmd := &cobra.Command{
+		Use:   "add",
+		Short: "Add",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if orgName == "" && projectName == "" {
+				return errors.New("either organization or project has to be specified")
+			}
+			if orgName != "" && projectName != "" {
+				return errors.New("only one of organization or project has to be specified")
+			}
+
+			client, err := cmdutil.Client(cfg)
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			if email == "" {
+				email = cmdutil.MandatoryInputPrompt("Please enter the email of the user.")
+			}
+			if role == "" {
+				role = cmdutil.MandatoryInputPrompt("Please enter the user's role.")
+			}
```

Comment:
Consider refactoring to a `cmdutil.StringPromptIfEmpty(&email, "Enter email of user")` helper. I guess this will be handy many places for getting missing required args.